server/src/main/scala/services/graphstages/VoyageManifestsGraphStage.scala

```diff
@@ -136,42 +141,39 @@ class VoyageManifestsGraphStage(bucketName: String, portCode: String, initialLas
 
   def fileNameAndContentFromZip[X](zipFileName: String,
                                    zippedFileByteStream: Source[ByteString, X],
-                                   maybePort: Option[String],
-                                   maybeAirlines: Option[List[String]]): Seq[(String, Option[VoyageManifest])] = {
+                                   port: String,
+                                   maybeAirlines: Option[List[String]]): List[(String, Option[VoyageManifest])] = {
```

Comment:
Do we possibly have a missing test here?

Reply:
Yes probably.

I'll try to refactor this whole thing to make it testable with a mocked stream of zip files